scripts/unit/npm-release-spec.js

```diff
@@ -226,109 +229,250 @@ const semanticReleaseNew = () => {
 }
 
 describe('semantic release', () => {
-  it('ends with no output if triggered by a pull request', () => {
-    const { currentVersion, nextVersion } = parseSemanticReleaseOutput(semanticReleasePullRequest)
+  let parseSemanticReleaseOutput
+  let releasePackages
+  let execaStub
 
-    la(currentVersion === undefined, 'Expected current version to be', undefined, 'but got', currentVersion, 'instead')
-    la(nextVersion === undefined, 'Expected current version to be', undefined, 'but got', nextVersion, 'instead')
-  })
-
-  describe('parses old version number when there are no updates', () => {
-    it('works with standard version number', () => {
-      const version = '1.2.3'
+  beforeEach(() => {
+    sinon.restore()
 
-      const { currentVersion, nextVersion } = parseSemanticReleaseOutput(semanticReleaseNoUpdate(version))
+    execaStub = sinon.stub()
 
-      la(currentVersion === version, 'Expected current version to be', version, 'but got', currentVersion, 'instead')
-      la(nextVersion === undefined, 'Expected next version to be', undefined, 'but got', nextVersion, 'instead')
+    const npmRelease = proxyquire('../npm-release', {
+      'execa': execaStub,
     })
 
-    it('works with version 0.x.x', () => {
-      const version = '0.0.1'
+    parseSemanticReleaseOutput = npmRelease.parseSemanticReleaseOutput
+    releasePackages = npmRelease.releasePackages
+  })
 
-      const { currentVersion, nextVersion } = parseSemanticReleaseOutput(semanticReleaseNoUpdate(version))
+  context('#parseSemanticReleaseOutput', () => {
```

Comment:
A lot of this is just indentation from grouping the previous tests together in a `context()`. View [it with whitespace ignored](https://github.com/cypress-io/cypress/pull/24017/files?w=1) for the true diff.